src/components/Modals/IframeModal.tsx

```diff
@@ -0,0 +1,37 @@
+import { Center, Modal, ModalCloseButton, ModalContent, ModalOverlay } from '@chakra-ui/react'
+import { useModal } from 'hooks/useModal/useModal'
+
+type IframeModalProps = {
+  url: string
+  title: string
+  width?: string
+  height?: string
+}
+
+export const IframeModal: React.FC<IframeModalProps> = ({
+  title,
+  url,
+  width = '482px',
+  height = '660px',
+}) => {
+  const { iframe } = useModal()
+  const { close, isOpen } = iframe
```

Comment:
Nitpickish but will make linters happy
```suggestion
  const { close: handleClose, isOpen } = iframe
```